components/tyk-gateway/values.yaml

```
@@ -156,6 +156,11 @@ gateway:
   # The port which will be exposed on the container for tyk-gateway
   containerPort: 8080
 
+  # By default, APIs, policies, and middleware are stored on a volume mount,
+  # enable to instead store them in the container's filesystem
+  detachInitContainer:
```

@jakub-bochenski `detachInitContainer.enabled` shoulds bit confusing.
Instead can we rename this field to just `disableInitContainer` without `enabled` as subfield. 
```suggestion
  disableInitContainer: false
```

---

I've already renamed it once per @buraksekili  suggestion, and then added it to umbrella charts.
I hope you made up your mind by now at least.

It's not very nice given that this PR has been waiting for over a month now

---

renamed to disableInitContainer